extensions/sql-bindings/src/common/telemetry.ts

```diff
@@ -48,6 +50,10 @@ export enum CreateAzureFunctionStep {
 	promptForIncludePassword = 'promptForIncludePassword',
 }
 
+export enum AzureFunctionUtils {
```

Comment:
This name is confusing - how is this an "Azure Function Utility"? Shouldn't `addSqlNugetPackage` just go under the `TelemetryActions` enum?